geostore/models.py

```diff
@@ -390,6 +391,17 @@ def get_property_title(self, prop):
 
         return prop
 
+    def get_property_type(self, prop):
+        """ Get json property type with its name """
+        json_form_properties = self.schema.get('properties', {})
+
+        if prop in json_form_properties:
+            data = json_form_properties[prop]
+            prop_type = data.get('type')
+            return prop_type
+
+        return prop
```

Comment:
The behavior returning the prop name instead of a type seems weird, how is it explained ?

Reply:
yes. bad copy / paste. thx

Reply:
Isn't a test for this case missing ?

Reply:
done